mmv1/products/netapp/Volume.yaml

```diff
@@ -503,3 +503,29 @@ properties:
     description: |
       Optional. Flag indicating if the volume will have an IP address per node for volumes supporting multiple IP endpoints.
       Only the volume with largeCapacity will be allowed to have multiple endpoints.
+  - !ruby/object:Api::Type::String
+    name: 'coldTierSizeGib'
+    description: |
+      Output only. Size of the volume cold tier data in GiB.
+    output: true
+  - !ruby/object:Api::Type::NestedObject
+    name: tieringPolicy
+    description: |-
+      Tiering policy for the volume.
+    update_mask_fields:
```

Comment:
non-actionable: I'll flag that requiring this means an AIP is being broken, as the top-level field should have been sufficient: https://google.aip.dev/161#specifying-specific-fields